lib/exercism/user_statistics.rb

```diff
@@ -11,7 +11,7 @@ def self.by_username(user)
   end
 
   def self.problem_completion(user, track)
-    user.exercises.where(language: track.id).map do |exercise|
+    user.exercises.current.where(language: track.id).map do |exercise|
```

Comment:
If you use `current` then it will exclude archived exercises, which are exercises that you should get credit for having submitted a solution to.